alert-receiver/action.yml

```
@@ -0,0 +1,53 @@
+name: Send alert to alertmanager-receiver
+description: This will send a json message to the alert manager receiver, formatting it, and then sending to Keybase.
+inputs:
+  status:
+    description: 'Options are: "firing, resolves, warning, message'
+    required: true
+  title:
+    description: 'Title of the Alert'
+    required: true
+  description:
+    description: 'Description of the Alert'
+    required: true
+  webhook_endpoint:
+    description: 'Routes to endpoints. Use the alertmanager-receiver URL with the additional route. Options are: "ips-alets, ips-info, teambottesting, devrel, ent-wallet"'
```

```suggestion
    description: 'Routes to endpoints. Use the alertmanager-receiver URL with the additional route.'
```

will likely change over time. Rather just leave them out